micromamba/src/env.cpp

```diff
@@ -227,6 +228,21 @@ set_env_command(CLI::App* com, Configuration& config)
                         channels.insert(chan.display_name());
                     }
                 }
+
+                // Add a `pip` subsection in `dependencies` listing wheels installed from PyPI
+                if (pip_versions_map.size() > 0)
+                {
+                    dependencies << (first_dependency_printed ? ",\n" : "") << "     { \"pip\": [\n";
+                    first_dependency_printed = false;
+                    for (const auto& [k, v] : pip_versions_map)
+                    {
+                        dependencies << (first_dependency_printed ? ",\n" : "") << "      \""
+                                     << v.name << "==" << v.version << "\"";
+                        first_dependency_printed = true;
+                    }
+                    dependencies << "\n    ]\n    }";
```

Comment:
Consider using '\t' instead of 4 spaces.

Reply:
conda is using spaces over tabs everywhere (even for 4 spaces), so I guess we better stick with them not to mix outputs with spaces and tabs. What do you think?

Reply:
Ah indeed, I thought we were already using "\t" everywhere else. I don't think that would result in mixing tabs and spaces though, since we are not appending anything to output produced by conda, but yeah, let's keep the codebase consistent and use 4 spaces for now.